Source/Scene/Scene.js

```diff
@@ -822,6 +822,19 @@ define([
         },
 
         /**
+         * Gets an event that's raised when the terrain provider is changed
+         * @memberof Scene.prototype
+         *
+         * @type {Event}
+         * @readonly
+         */
+        terrainProviderChanged : {
```

Comment:
I mean will anyone use it on the `Globe` directly?  Or will everyone use it through `Scene`?  If the later, then why not keep the API surface area small and not provide two ways to do the same thing?